app/bundles/LeadBundle/Model/CompanyModel.php

```diff
@@ -516,6 +516,62 @@ public function removeLeadFromCompany($companies, $lead): void
         unset($lead, $deleteCompany, $companies);
     }
 
+    /**
+     * Get a list of companies with names only.
+     *
+     * @param mixed[]|string $filter
+     * 
+     * @return string[]
+     */
+    public function getSimpleLookupResults(string $type, array|string $filter = '', int $limit = 10, int $start = 0, string $exclude = ''): array
```

Comment:
Tests are failing with
```php
TypeError: Mautic\LeadBundle\Model\CompanyModel::getSimpleLookupResults(): Argument #5 ($exclude) must be of type string, null given
```
```suggestion
    public function getSimpleLookupResults(string $type, array|string $filter = '', int $limit = 10, int $start = 0, ?string $exclude = ''): array
```